Framework/Kernel/inc/MantidKernel/NexusHDF5Descriptor.h

```diff
@@ -12,8 +12,7 @@
 #include <set>
 #include <string>
 
-namespace Mantid {
-namespace NeXus {
+namespace Mantid::Kernel {
 
 class DLLExport NexusHDF5Descriptor {
```

Comment:
`DLLExport` -> `MANTID_KERNEL_DLL`. You'll need to swap `#include "MantidKernel/System.h"` for `#include "MantidKernel/DllConfig.h"`